ts/packages/node/src/stores/IniFile.ts

```diff
@@ -0,0 +1,55 @@
+import { promises as fs } from 'fs';
+import _ from 'lodash';
+import ini from 'ini';
+import { ConfigStore, ConfigStoreQuery, Config } from '@configu/ts';
+
+export type IniFileConfigStoreConfiguration = { path: string };
+
+export class IniFileConfigStore extends ConfigStore {
+  private readonly path: string;
+  constructor({ path }: IniFileConfigStoreConfiguration) {
+    super('ini-file');
+    this.path = path;
+  }
+
+  async read(): Promise<Config[]> {
+    const data = await fs.readFile(this.path, 'utf8');
+    const iniObject = ini.parse(data);
+
+    return Object.entries(iniObject).flatMap(([set, keyValuePairs]) =>
+      Object.entries(keyValuePairs)
+        .filter(([, value]) => typeof value === 'string' && !Array.isArray(value))
+        .map(([key, value]) => ({ set, key, value: value as string })),
+    );
```

Comment:
We need to adjust how we treat the `set`, any key-value pair that don't belong to a section will have the key converted to a set which isn't accurate.

Here is a suggested implementation:
```suggestion
    return Object.entries(iniObject).flatMap(([key, value]) => {
      if (typeof value === 'string') {
        return { set: '', key, value };
      }
      if (_.isPlainObject(value)) {
        return _(Object.entries(value))
          .filter(([, innerValue]) => typeof innerValue === 'string')
          .map(([innerKey, innerValue]) => ({
            set: key,
            key: innerKey,
            value: innerValue as string,
          }))
          .value();
      }
      return [];
    });
```

Not so sure about why you decided to add `!Array.isArray(value)` explicitly, please shed some light on it.